schema.md

```diff
@@ -56,3 +57,20 @@ Place: {
     }
 }
 ```
+
+## Event
+
+```
+Event: { 
+    coordinates: {
+        lat: #,
+        lng: #
+    },
+    description: "", # Used for badging
```

Comment:
The UX mocks (via https://github.com/mozilla-mobile/prox/issues/99) show text, followed by an underlined text acting as a CTA to click for more info. I think we should separate description into an array (or two keys), where the first part is the normal text and the second part is the underlined text.